download-data.py

```diff
@@ -243,30 +249,36 @@ def main() -> None:
     # Always include json, tsv metadata files and DATA_USAGE.md
     patterns = ["*.json", "*.tsv", "DATA_USAGE.md"]
 
-    if args.include_reports:
-        patterns += ["*.html"]
-
-    if "sce" in formats:
-        patterns += [f"*_{level}.rds" for level in includes]
+    # Only include other files if metadata_only is not false
+    # If metadata_only is true, overwrite other defaults for Download Summary printing
+    if args.metadata_only:
+        formats = ["N/A"]
+        includes = ["N/A"]
+    else:
```

Comment:
For the future revision: I would rather have a guard that `--metadata_only` and other `--include` options are exclusive. That will allow you to avoid the `else:` and changing nesting levels.

I would also expect people might want to download metadata for only specific projects, so that part of the code would not be nested, though you would also want to guard against choosing samples, since w don't have individual metadata files.